standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/jdbc/ParameterizedBatchCommand.java

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
maybe it's just a matter taste, but I feel that an interface should go into the same package where it's typically implemented by many classes, if that package is typically about to contain those particular classes, so instead of having org.apache.hadoop.hive.metastore.txn.jdbc.ParameterizedBatchCommand and its subclasses in org.apache.hadoop.hive.metastore.txn.impl.commands, all of them can go into a package "org.apache.hadoop.hive.metastore.txn.command"

what do you think?

and the same applies to functions and other stuff introduced in this PR

Reply:
I was thinking the other way around: These interfaces and abstract classes are parameters in the various overloads of `MultiDataSourceJdbcResource.execute()` methods. Therefore I thought these should be in the same package as these classes together form the "API" to use by `[Compaction]TxnHandler` function implementations. However, I'm not strictly tied to this idea, as it doesn't really matter in which of these packages we put them.